sources/packages/web/src/composables/useFormioUtils.ts

```diff
@@ -261,6 +280,32 @@ export function useFormioUtils() {
     customUtils[name] = method;
   };
 
+  /**
+   * Generate the dynamic form definition removing unwanted properties.
+   */
+  const getReadyToSaveFormDefinition = (formDefinition: unknown): unknown => {
```

Comment:
Please add @parm and @returns to the comments.

Reply:
Added.